spec/tictactoe_clojure/computer_player_spec.clj

```diff
@@ -0,0 +1,33 @@
+(ns tictactoe-clojure.computer-player-spec
+  (:require [speclj.core :refer :all]
+            [tictactoe-clojure.computer-player :refer :all]))
+
+(describe "tictactoe-clojure.computer-player"
+  (with x-win-1st-row-board ["X" "X" "X"  4 5 6 7 8 9])
+  (with o-on-1-board ["O" 2 3 4 5 6 7 8 9])
+  (with x-1-move-from-win ["X" "X" 3 "O" "O" 6 7 8 9])
+  (with o-1-move-from-win ["X" 2 3 "O" "O" 6 7 "X" 9])
+  (with o-win-2nd-row-board ["X" 2 "X" "O" "O" "O" 7 8 9])
+  (with board-with-draw ["X" "O" "X" "O" "O" "X" "X" "X" "O"])
+
+  (describe "#optimal-move-by-score"
+    (it "return vector move 3 score 9 if X turn and it is one move away from winning"
+      (should= '(3 9) (optimal-move-by-score "X" @x-1-move-from-win 1 true)))
+    (it "return vector move 6 score 0 if X turn and it blocks O from winning"
+      (should= '(6 0) (optimal-move-by-score "X" @o-1-move-from-win 1 true))))
+
+  (describe "#score-board"
+    (it "returns 0 if it is a draw"
+      (should= 0 (score-board "X" @board-with-draw 0 true)))
+    (it "returns 10 if Xs are on the winning combinations"
+      (should= 10 (score-board "X" @x-win-1st-row-board 0 true)))
+    (it "returns -10 if Os are on the winning combinations"
+      (should= -10 (score-board "X" @o-win-2nd-row-board 0 true))))
```

Comment:
Yes, this should be a private function because it shouldn't be callable outside of the computer player.